scripts/apollo_ci.sh

```diff
@@ -29,8 +32,12 @@ function main() {
     if [ "$1" == "test" ]; then
         info "Running CI Test ..."
         run_ci_test
-    else
+    elif [ "$1" == "build" ]; then
```

Comment:
Since local var `cmd` was defined, better use `cmd` rather than the original `$1`